File: Basics/Conditional Statements Advanced/Exercise/07.py
month = input()
count_night = int(input())

price_studio = 0
price_apartment = 0
price_of_entire_stay_in_studio = 0
price_of_entire_stay_in_apartment = 0

if month == "May" or month == "October":
    price_studio = 50
    price_apartment = 65
    if 7 < count_night < 14:
        price_studio = price_studio * 0.95
    elif count_night > 14:
        price_studio = price_studio * 0.70
        price_apartment = price_apartment * 0.9


elif month == "June" or month == "September":
    price_studio = 75.2
    price_apartment = 68.70
    if count_night > 14:
        price_studio = price_studio * 0.80
        price_apartment = price_apartment * 0.9

elif month == "July" or month == "August":
    price_studio = 76
    price_apartment = 77
    if count_night > 14:
        price_apartment = price_apartment * 0.9

price_of_entire_stay_in_studio = count_night * price_studio
price_of_entire_stay_in_apartment = count_night * price_apartment

print(f"Apartment: {price_of_entire_stay_in_apartment:.2f} lv.")
print(f"Studio: {price_of_entire_stay_in_studio:.2f} lv.")

File: JS-Basics/Conditional Statements Advanced/Exercise/07.js
function calculateStayCost(month, countNights) {
    let priceStudio = 0;
    let priceApartment = 0;
    let priceOfEntireStayInStudio = 0;
    let priceOfEntireStayInApartment = 0;

    if (month === "May" || month === "October") {
        priceStudio = 50;
        priceApartment = 65;
        if (countNights > 7 && countNights < 14) {
            priceStudio *= 0.95;
        } else if (countNights > 14) {
            priceStudio *= 0.70;
            priceApartment *= 0.9;
        }
    } else if (month === "June" || month === "September") {
        priceStudio = 75.2;
        priceApartment = 68.7;
        if (countNights > 14) {
            priceStudio *= 0.80;
            priceApartment *= 0.9;
        }
    } else if (month === "July" || month === "August") {
        priceStudio = 76;
        priceApartment = 77;
        if (countNights > 14) {
            priceApartment *= 0.9;
        }
    }

    priceOfEntireStayInStudio = countNights * priceStudio;
    priceOfEntireStayInApartment = countNights * priceApartment;

    console.log(`Apartment: ${priceOfEntireStayInApartment.toFixed(2)} lv.`);
    console.log(`Studio: ${priceOfEntireStayInStudio.toFixed(2)} lv.`);
}
